domains/transfer/Testing/Testcontainers/PostgresContainer.cs

```diff
@@ -1,10 +1,9 @@
 using System.Threading.Tasks;
 using Testcontainers.PostgreSql;
-using Xunit;
 
 namespace Testing.Testcontainers;
 
-public class PostgresContainer : IAsyncLifetime
+public class PostgresContainer
```

Comment:
This is used as test fixture, I believe this should implement IAsyncLifetime in order to clean up resources.

Reply:
That would add back xUnit as dependency. Then we are back where we started